src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,137 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void registerUserNull_NotOk() {

```

Comment:
Use this pattern  for naming for all tests
![image](https://github.com/mate-academy/jv-registration-validation/assets/63095184/c47ea891-162e-46c4-b14e-d91a73a5036d)